Source/WebCore/page/PointerCaptureController.cpp

```diff
@@ -384,16 +385,24 @@ void PointerCaptureController::pointerEventWillBeDispatched(const PointerEvent&
     if (!isPointerdown && !isPointerup)
         return;
 
+    auto handlePopoverLightDismiss = [&]() {
+        if (auto* element = dynamicDowncast<HTMLElement>(target))
```

Comment:
I realize this check isn't correct, we should just pass any target directly into `handlePopoverLightDismiss`. Light dismiss should happen when you click outside on any node, not just HTML elements.